### PR TITLE
Update JUnit Platform 1.0.0-RC2 → 1.0.0, JUnit Jupiter 5.0.0-RC2 → 5.0.0.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Release Notes http://junit.org/junit5/docs/current/user-guide/#release-notes-5.0.0

### CODE REVIEW

File: spek-junit-platform-engine/src/main/kotlin/org/jetbrains/spek/engine/SpekTestEngine.kt [hunk 1]
```diff
@@ -41,7 +41,7 @@ class SpekTestEngine: HierarchicalTestEngine<SpekExecutionContext>() {
     val defaultInstanceFactory = object: InstanceFactory {
         override fun <T: Spek> create(spek: KClass<T>): T {
             return spek.objectInstance ?: spek.constructors.first { it.parameters.isEmpty() }
-                .call()
+                    .call()
```
Comment (raniejade) [inline spek-junit-platform-engine/src/main/kotlin/org/jetbrains/spek/engine/SpekTestEngine.kt]: Minor stuff, you can ignore this. We should probably agree on the code style at some point.

Comment (artem-zinnatullin) [reply spek-junit-platform-engine/src/main/kotlin/org/jetbrains/spek/engine/SpekTestEngine.kt]: I've tried :( IntelliJ just does that, I can edit it in text editor if you want

Comment (raniejade) [reply spek-junit-platform-engine/src/main/kotlin/org/jetbrains/spek/engine/SpekTestEngine.kt]: You just need to set your trailing indent to 4 spaces in the coding style in IDEA, I think the default is 8.

**Comment by artem-zinnatullin** [comment]
Formatting foxed!

**Pull request merged**